recipe/build.sh

```diff
@@ -6,28 +6,34 @@ elif [[ $(uname) == Linux ]]; then
   export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
 fi
 
-export LDFLAGS="$LDFLAGS -L$PREFIX/lib -Wl,-rpath,$PREFIX/lib"
+export LDFLAGS="$LDFLAGS -L$PREFIX/lib -Wl,-rpath,$PREFIX/lib -lcurl -lhdf5 -lhdf5_hl -ldf -lmfhdf"
 export CFLAGS="$CFLAGS -fPIC -I$PREFIX/include"
+if [[ $(uname) == Darwin ]] && [[ "${CC}" != "clang" ]]; then
+  export FFLAGS="-isysroot $CONDA_BUILD_SYSROOT $FFLAGS"
+fi
+
+# This really mucks with the build.
+rm -rf ${PREFIX}/lib/cmake/netCDF/*
 
 # Build static.
 mkdir build_static && cd build_static
-cmake -D CMAKE_INSTALL_PREFIX=$PREFIX \
-      -D CMAKE_INSTALL_LIBDIR:PATH=$PREFIX/lib \
-      -D BUILD_SHARED_LIBS=OFF \
+cmake -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_INSTALL_LIBDIR:PATH=$PREFIX/lib \
+      -DBUILD_SHARED_LIBS=OFF \
       $SRC_DIR
 
 make
 # ctest  # Run only for the shared lib build to save time.
 make install
 
 make clean
-
 cd ..
-mkdir build_shared && cd build_shared
+
 # Build shared.
-cmake -D CMAKE_INSTALL_PREFIX=$PREFIX \
-      -D CMAKE_INSTALL_LIBDIR:PATH=$PREFIX/lib \
-      -D BUILD_SHARED_LIBS=ON \
```

Comment:
Is the removal in the space there really needed? (Kind of makes it hard to review.)

Reply:
Ohhh sorry. I didn't like the spaces. I will put them back.

Reply:
No need. Just avoid doing that when sending so many commits. It makes it really hard to review. Leave cosmetic changes to simpler PRs.